packages/embark/src/lib/utils/solidity/remapImports.ts

```diff
@@ -124,19 +138,31 @@ const isHttp = (input: string) => {
 };
 
 const replaceImports = (remapImports: RemapImport[]) => {
-  const byPath: {[path: string]: [{searchValue: string, replaceValue: string}]} = groupBy(remapImports, "path");
+  const byPath: { [path: string]: [{ remapping: ImportRemapping }] } = groupBy(remapImports, "path");
   Object.keys(byPath).forEach((p) => {
     let source = fs.readFileSync(p, "utf-8");
-    byPath[p].forEach(({searchValue, replaceValue}) => {
-      source = source.replace(`import "${searchValue}"`, `import "${replaceValue}"`);
+    byPath[p].forEach(({ remapping }) => {
+      source = source.replace(`import "${remapping.prefix}"`, `import "${remapping.target}"`);
     });
     fs.writeFileSync(p, source);
   });
 };
 
+const addRemappingsToFile = (file: File, remapImports: RemapImport[]) => {
+  const byPath: { [path: string]: [{ remapping: ImportRemapping }] } = groupBy(remapImports, "path");
+  const paths = Object.keys(byPath);
+  if (paths) {
+    file.importRemappings = []; // clear as we already have the first remapping added
+    paths.forEach((p) => {
+      const [...remappings] = byPath[p].map((importRemapping) => importRemapping.remapping);
```

Comment:
Can't you just do `const remappings = ...` since `.map` returns an array

Reply:
Updated.